File: stroke-prediction/src/app/core/model/record.dto.ts
export class RecordDto {
  id: number | null;
  gender: number | null;
  age: number | null;
  bmi: number | null;
  hypertension: boolean | null;
  heartDisease: boolean | null;
  avgGlucoseLevel: number | null;
  smokingStatus: number | null;
  stroke: boolean | null;
  similarity: number | null;

  constructor() {
    this.id = null;
    this.gender = null;
    this.age = null;
    this.hypertension = null;
    this.heartDisease = null;
    this.avgGlucoseLevel = null;
    this.bmi = null;
    this.smokingStatus = null;
    this.stroke = null;
    this.similarity = null;
  }
}

File: stroke-prediction-server/src/main/java/com/strokepredictionserver/model/RecordDto.java
package com.strokepredictionserver.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RecordDto {

    private Integer id;

    private Integer gender;

    private Integer age;

    private Boolean hypertension;

    private Boolean heartDisease;

    private Double avgGlucoseLevel;

    private Double bmi;

    private Integer smokingStatus;

    private Boolean stroke;

    private Double similarity;

}
